manual/cypher/cypher-docs/src/docs/graphgists/intro/data-structures.adoc

```diff
@@ -1,16 +1,43 @@
 = Utilizing Data Structures
 
+//file:movies.csv
+//file:roles.csv
+//file:persons.csv
+//file:movie_actor_roles.csv
```

Comment:
Yes, we abuse comment syntax for directives.
We should find a better syntax for this, at some point.